src/tools/tests/allocator-recipe-resolver-test.ts

```diff
@@ -1229,7 +1398,7 @@ policy Policy0 {
   }
 }
 policy Policy1 {
-  @allowedRetention(medium: 'Ram', encryption: true)
+  @allowedRetention(medium: 'Ram', encryption: false)
```

Comment:
could you, please, explain why these capabilities needed to change?

Reply:
With encryption, there was no storage driver and allocator was not able to create a storage key.